### PR TITLE
project: update to sdk structure

### PR DESCRIPTION
**Pull request by erizocosmico**

### CODE REVIEW

File: driver/main.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	fmt.Printf("version: %s\nbuild: %s\n", version, build)
+
+	_, err := os.Stat(sdk.NativeBin)
+	if err == nil {
+		fmt.Println("native: ok")
+		return
+	}
+
+	fmt.Printf("native: %s\n", err)
+}
```
Comment (ajnavarro) [inline driver/main.go]: go fmt?

**Comment by erizocosmico** [comment]
idk, it was autogenerated by the sdk

El mar., 21 mar. 2017 a las 12:27, Antonio Navarro Perez (<notifications@github.com<mailto:notifications@github.com>>) escribió:

@ajnavarro commented on this pull request.

________________________________

In driver/main.go<https://github.com/bblfsh/clojure-driver/pull/2#discussion_r107130639>:

> +)
+
+var version string
+var build string
+
+func main() {
+       fmt.Printf("version: %s\nbuild: %s\n", version, build)
+
+       _, err := os.Stat(sdk.NativeBin)
+       if err == nil {
+               fmt.Println("native: ok")
+               return
+       }
+
+       fmt.Printf("native: %s\n", err)
+}


go fmt?

**Comment by smola** [comment]
@erizocosmico You have to update the SDK, that autogenerated main.go is outdated.

**Comment by erizocosmico** [comment]
oh damn, merged it by mistake

**Pull request merged**